packages/as-transformer/src/helpers/protobuf.ts

```diff
@@ -0,0 +1,125 @@
+enum ProtoType {
+  Double = 'double',
+  Float = 'float',
+  Int32 = 'int32',
+  Int64 = 'int64',
+  UInt32 = 'uint32',
+  UInt64 = 'uint64',
+  SInt32 = 'sint32',
+  SInt64 = 'sint64',
+  Fixed32 = 'fixed32',
+  Fixed64 = 'fixed64',
+  SFixed32 = 'sfixed32',
+  SFixed64 = 'sfixed64',
+  Bool = 'bool',
+  String = 'string',
+  Bytes = 'bytes',
+}
+
+interface FieldSpec {
+  type: ProtoType;
+  repeated: boolean;
+}
+
+export interface Argument {
+  name: string;
+  type: string;
+}
+
+export function generateProtoFile(
+  name: string,
+  args: Argument[],
+  returnedType: string,
+): string {
+  const argumentMessages = args.map((arg, index) =>
+    generateArgumentMessage(arg, index + 1),
+  );
+  const fields = argumentMessages.join('\n');
+
+  let protoFile = `syntax = "proto3";
+
+message ${name} {
+${fields}
+}`;
+
+  if (returnedType != '' && returnedType != 'null') {
```

Comment:
I don't know when we have 'null' as return type

```suggestion
  if (returnedType && returnedType !== 'void') {
```

Reply:
see https://www.assemblyscript.org/examples/snippets.html#handling-null

Reply:
Thanks for the link. Shouldn't we add void ?